nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/api/AbstractSession.java

```diff
@@ -1414,7 +1414,7 @@ protected void removeNotifyOneDoc(Document doc) {
                 DocumentModel sourceDocModel = readModel(sourceDoc);
                 if (sourceDocModel != null) {
                     options.put("comment", versionLabel); // to be used by
-                                                          // audit
+                    // audit
```

Comment:
Maybe clearer on a single line?
```suggestion
options.put("comment", versionLabel); // to be used by audit service
```